answerking_app/views/product_views.py

```diff
@@ -92,8 +92,9 @@ def post(self, request: Request, *args, **kwargs) -> Response:
 
 class ProductDetailView(
     RetireMixin,
-    generics.UpdateAPIView,
+    mixins.UpdateModelMixin,
```

Comment:
changed because generics.UpdateAPIView was also creating a PATCH endpoint in the swagger-UI contract